packages/unmock-core/src/generator.ts

```diff
@@ -148,28 +148,31 @@ const chooseResponseFromOperation = (
   }
 
   const response = responses[chosenCode as keyof Responses];
-  if (
-    response === undefined ||
-    isReference(response) ||
-    response.content === undefined
-  ) {
+  if (response === undefined) {
+    throw new Error("Not sure what went wrong");
```

Comment:
Maybe a real error message like `Got undefined response` would be easier to debug?

Reply:
:+1: I can change to that. Many of those are mostly for type-checking, but I don't see why e.g. `const response = responses[chosenCode as keyof Responses]` will ever be undefined when `chosenCode` is defined and chosen from the keys of `responses`, etc.